Разметка всех страниц

Lyezik opened the pull request:
---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/34/user/2097541/tasks/4)

:boom: https://htmlacademy-htmlcss.github.io/2097541-sedona-34/2/

Lyezik committed:
Добавляет разметку шапки главной страницы

Lyezik committed:
Update index.html

1. Вынес Логотип из списка
2. Кнопку "Хочу сюда!" изменил на ссылку, как в видео-гайде

Lyezik committed:
Добавляет секцию приветствия и преимуществ

Lyezik committed:
Добавляет секцию поиска отеля и исправляет некоторые кнопки

1. Кнопка "Хочу сюда!" в шапке исправлена с ссылки на кнопку
2. Добавлены классы в секции welcome
3. Код отформатирован

Lyezik committed:
Добавляет секцию рассылки писем

Lyezik committed:
Добавляет подвал сайта

Lyezik committed:
Переносит раздел рассылки писем в подвал и добавляет заголовки к секциям подвала

Lyezik committed:
Убирает из разметки текстовые картинки из раздела welcome и иконки в списке преимуществ

Lyezik committed:
Добавляет класс btn всем кнопкам

Lyezik committed:
Убирает класс для скрытия с кнопки подписки на рассылку

Lyezik committed:
Изменяет секцию welcome на div

т.к. там только картинка и смысловой составляющей нет

Lyezik committed:
Делает список преимуществ как список в списке и добавляет обертку элементам списка с картинкой, чтобы отделить текст от картинки

Lyezik committed:
Повышает уровень заголовков у преимуществ города Седоны

Lyezik committed:
Меняет название обертки приветственной картинки и делает имя формы отправки почты

Lyezik committed:
Добавляет скрытую подпись "Email" перед формой

Lyezik committed:
Оборачивает label и input в тег p (в форме отправки Email)

Lyezik committed:
Добавляет ссылки на соц сети и сайт академии

Lyezik committed:
Форматирование кода

alex-koshara commented on index.html:
```diff
@@ -1,12 +1,167 @@
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Седона</title>
```
Можно в целом не указывать что это HTML Academy здесь, так как это сайт Седоны

alex-koshara commented on index.html:
```diff
@@ -1,12 +1,167 @@
+          </a>
+        </li>
+        <li class="navigation-item">
+          <input class="btn btn-brown navigation-btn" type="button" value="Хочу сюда!">
```
Давай для кнопок использовать кнопки. Инпуты с типом кнопка - это довольно устаревший вариант :)

alex-koshara commented on index.html:
```diff
@@ -1,12 +1,167 @@
+      <h2 class="search-hotel-title">Заинтересовались?</h2>
+      <p class="search-hotel-description">Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения
+        гостиниц в Седоне</p>
+      <input class="btn btn-brown search-hotel-btn" type="button" value="Поиск гостиницы в седоне">
```
Здесь тоже используем `<button>`

alex-koshara commented on index.html:
```diff
@@ -1,12 +1,167 @@
+          </label>
+          <input class="field" placeholder="Ваш e-mail" type="email" name="user-email" id="newsletter-email" required>
+        </p>
+        <button class="btn btn-blue newsletter-btn" type="submit">
```
Круто, что используешь верный тип для кнопки!

alex-koshara commented on index.html:
```diff
@@ -1,12 +1,167 @@
+          <input class="field" placeholder="Ваш e-mail" type="email" name="user-email" id="newsletter-email" required>
+        </p>
+        <button class="btn btn-blue newsletter-btn" type="submit">
+          <span class="newsletter-btn-subscribe">Подписаться</span>
```
Здесь `span` избыточен, вполне можно будет без него обойтись

alex-koshara commented on index.html:
```diff
@@ -1,12 +1,167 @@
+
+    <section class="footer-social">
+      <h2 class="footer-social-title">
+        <span class="visually-hidden">Ссылки на социальные сети</span>
```
Здесь тоже span избыточен, если в заголовке будет только скрытый текст. Можно сразу сделать так:
`<h2 class="visually-hidden">title</h2>`

alex-koshara commented on index.html:
```diff
@@ -1,12 +1,167 @@
+        <span class="visually-hidden">Контактные данные</span>
+      </h2>
+      <address class="footer-contacts-address">
+        <a class="footer-contacts-address-phone" href="tel:+78128121212">+7 (812) 812-12-12</a>
```
Круто, что использовал ссылку на телефон

alex-koshara reviewed (commented):
Хочу так же подметить, что классно поработал с коммитами и их названиями

Lyezik committed:
Исправляет ошибки, указанные наставником

1. Кнопки реализованы через <button>, а не через <input>
2. Убран лишний span внутри кнопки Подписаться
3. Невидимый заголовок реализован без span

Lyezik committed:
Убирает ссылку, ведущую в раздел "О Седоне", т.к. такой страницы в макете нет

По требованиям к ДЗ

Lyezik committed:
Добавляет указатель шапку, показывающий на какой странице находится пользователь

Lyezik committed:
Создаёт шапку и подвал

Lyezik committed:
Добавляет шапку основной части каталога

Lyezik committed:
Добавляет фильтры гостиниц в каталог

Lyezik committed:
Изменяет надпись "Главная" в хлебных крошках на картинку-ссылку

Lyezik committed:
Убирает ненужный класс из ссылки на главную страницу в хлебн. крошках

alex-koshara commented on catalog.html:
```diff
@@ -0,0 +1,173 @@
+<body>
+  <header class="page-header">
+    <nav class="navigation">
+      <a class="logo-link" href="#">
```
Если копипастим - то перепроверяем что скопипастили 👿

alex-koshara commented on catalog.html:
```diff
@@ -0,0 +1,173 @@
+  <main class="main-catalog">
+    <header class="catalog-header">
+      <h1 class="catalog-header-title">Гостиницы Седоны</h1>
+      <ul class="breadcrumbs-list">
```
Хлебные крошки, это один из тех списков, который может быть упорядоченным (нумерованным)

alex-koshara commented on catalog.html:
```diff
@@ -0,0 +1,173 @@
+      </ul>
+    </header>
+
+    <section class="catalog-hotels">
```
Круто что используешь семантические теги!

Lyezik committed:
Исправляет ошибки

1. Добавляет адрес для ссылки логотипа Седоны
2. Делает список хлебных крошек упорядоченным

Lyezik committed:
Добавляет сортировку карточек

Lyezik committed:
Добавляет карточку гостиницы

Lyezik committed:
Форматирует код

alex-koshara commented on catalog.html:
```diff
@@ -135,19 +136,19 @@ <h2 class="visually-hidden">Список гостиниц с фильтрами<
+        <span class="hotel-card-price">От 4000 ₽</span>
+        <a class="btn btn-brown hotel-card-more" href="#">Подробнее</a>
+        <button class="btn btn-blue hotel-card-favorites-btn" type="button">В избранное</button>
+        <img class="hotel-card-stars-img" src="#" alt="Звезды для оценки гостиницы">
```
Если в одном случае заполнил `alt`для `img`, то давай тогда везде его заполнять :)

Lyezik committed:
Добавляет описание картинки карточки

Lyezik committed:
Добавляет остальные карточки и пагинацию

Lyezik committed:
Разбивает основную часть страницы на 2 секции

На фильтр и сортировку

Lyezik committed:
Форматирует код

alex-koshara commented on catalog.html:
```diff
@@ -135,20 +142,91 @@ <h2 class="visually-hidden">Список гостиниц с фильтрами<
       </a>
+      <ul class="hotel-list">
+        <li class="hotel-card">
+          <a class="hotel-card-link" href="#">
```
Карточка отеля - вполне может быть тегом `article`

alex-koshara commented on catalog.html:
```diff
@@ -135,20 +142,91 @@ <h2 class="visually-hidden">Список гостиниц с фильтрами<
-
-      </div>
+      </ul>
+      <ul class="pagination">
```
Вот здесь как раз нам важен порядок. Странно было бы видеть пагинацию из 1000 страниц где они могут идти в рандомном порядке. Тяжело будет найти нужную тогда

Lyezik committed:
Исправляет ошибки

1. Оборачивает карточки в article
2. Делает пагинацию упорядоченным списком

alex-koshara commented on catalog.html:
```diff
@@ -144,59 +144,67 @@ <h2 class="catalog-hotel-sort-title">
-          <button class="btn btn-blue hotel-card-favorites-btn" type="button">В избранное</button>
-          <img class="hotel-card-stars-img" src="#" alt="Звезды для оценки гостиницы">
-          <p class="btn rating-description">Рейтинг:<span class="rating-value">8,5</span></p>
+          <article>
```
Супер! Потом, возможно, сюда надо будет ещё класс добавить
